README.md

```diff
@@ -62,7 +89,41 @@ router.get('/', function (req, res) {
 });
 ```
 
-As it turns out, this middleware will do the above by default, so if you don't pass a store you will automagically get all of the properties from `res.locals` passed as props to your component.
+If passing a single container, this middleware will do the above by default, so if you don't pass a store you will automagically get all of the properties from `res.locals` passed as props to your component.
+
+If you pass jsx, you gain flexibility, but lose the 'automagical' convenience described above, and must pass props down directly.
+
+### Options
+
+```javascript
+// *
+// Generating middleware with config options
+// *
+router.use(reactExpressMiddleware({
+	 // [ Options and their defaults ]
```

Comment:
Personally, I just write the code with the defaults shown, like this:

```javascript
// defaults shown
router.use(reactExpressMiddleware({
  element: document.body, // The element on the front-end to render into, can be a selector (string)
  renderMethod: ReactDOM.render, // or ReactDOMServer.renderToString on the server
  template: 'index',  // template passed to express' render
  key: 'content' // the variable exposed to the express template engine with the rendered html string
)}
```

Secondly, since this is isomorphic, having a different option for the render method on server vs client is not great.  I didnt notice this in the other PR, but you should change it to just `renderMethod` and let the user decide if they want on what to do about handling isomorphism.